saleor/graphql/shop/resolvers.py

```diff
@@ -0,0 +1,35 @@
+from ...core.taxes import display_gross_prices
+from ...plugins.manager import get_plugins_manager
+from ...shipping.models import ShippingMethod
+from ..channel import ChannelContext
+
+
+def resolve_available_shipping_methods(channel_slug: str, address):
+    available = ShippingMethod.objects.filter(
+        channel_listings__channel__slug=channel_slug
+    )
+    if address and address.country:
+        available = available.filter(
+            shipping_zone__countries__contains=address.country,
+        )
+
+    if available is None:
+        return []
+    manager = get_plugins_manager()
```

Comment:
This makes a database query, while the manager is already prefetched and available as `info.context.plugins`. Since this function is called as a resolver we can pass the `info` object as param (or just the manager if it makes more sense).